File: 2/x.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Threading;



namespace Problem
{
    internal class Program
    {
        static void Main()
        {
            string find = "";
            string missing = "";
            Console.WriteLine("Enter complete sentence :");
            find = Console.ReadLine();
            Console.WriteLine("Enter missing vowels :");
            missing = Console.ReadLine();
            string complete = uncensor(find, missing);
            Console.WriteLine(complete);
        }
        static string uncensor(string find, string missing)
        {
            int j = 0;
            char[] findArray = find.ToCharArray();

            for (int i = 0; i < find.Length; i++)
            {
                if (findArray[i] == '*')
                {
                    findArray[i] = missing[j];

                    j++;
                }
            }
            find = new string(findArray);
            return find;
        }
    }
}

File: 3/x.cpp
#include <iostream>
#include <conio.h>
using namespace std;
string uncensor(string find, string missing);
int main()
{
    string find = "";
    string missing = "";
    cout << "Enter complete sentence :";
    getline(cin, find);
    cout << "Enter missing vowels :";
    getline(cin, missing);
    string complete = uncensor(find, missing);
    cout << complete;
    return 0;
}
string uncensor(string find, string missing)
{
    int j = 0;
    for (int i = 0; i < find.length(); i++)
    {
        if (find[i] == '*')
        {
            find[i] = missing[j];

            j++;
        }
    }
    return find;
}